app/views/leaderboard.scala.html

```diff
@@ -164,12 +165,119 @@ <h5>@Messages("leaderboard.weekly.detail")</h5>
             </div>
         </div>
     }
+    @if(user && user.get.role.getOrElse("") != "Anonymous" && !UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
+        <div class="item leaderboard-table">
+            @if(lang.code == "es") {
+                <h1 class="leaderboard-header">@Messages("leaderboard.org.title") @OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head)</h1>
```

Comment:
We should be able to remove this if/else and instead just pass in the org name as a parameter using `{}` in the messages file.